controller/backup_controller.go

```diff
@@ -299,6 +299,10 @@ func (bc *BackupController) reconcile(backupName string) (err error) {
 		// Disable monitor regardless of backup state
 		bc.disableBackupMonitor(backup.Name)
 
+		if backup.Status.State == longhorn.BackupStateError || backup.Status.State == longhorn.BackupStateUnknown {
+			bc.eventRecorder.Eventf(backup, corev1.EventTypeWarning, string(backup.Status.State), "Failed backup %s has been deleted: %s", backup.Name, backup.Status.Error)
```

Comment:
Is this event statement still true when failed-backup-ttl is disabled?

Reply:
Yes, I think it is still true, when users delete a failed backup manually and failed-backup-ttl is disabled.
Should we record this event only when failed-backup-ttl is enabled?